daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-template.md

```diff
@@ -146,6 +253,58 @@ The properties for the Multi-App Run template align with the `dapr run` CLI flag
+| `metricsPort`            | N        | The port that Dapr sends its metrics information to |  |
+| `unixDomainSocket`       | N        | Path to a unix domain socket dir mount. If specified, communication with the Dapr sidecar uses unix domain sockets for lower latency and greater throughput when compared to using TCP ports. Not available on Windows. | `/tmp/test-socket` |
+| `profilePort`            | N        | The port for the profile server to listen on |  |
+| `enableProfiling`        | N        | Enable profiling via an HTTP endpoint |  |
+| `apiListenAddresses`     | N        | Dapr API listen addresses |  |
+| `logLevel`               | N        | The log verbosity. |  |
+| `appMaxConcurrency`      | N        | The concurrency level of the application; default is unlimited |  |
+| `placementHostAddress`   | N        |  |  |
+| `appSSL`                 | N        | Enable https when Dapr invokes the application |  |
+| `daprHTTPMaxRequestSize` | N        | Max size of the request body in MB. |  |
+| `daprHTTPReadBufferSize` | N        | Max size of the HTTP read buffer in KB. This also limits the maximum size of HTTP headers. The default 4 KB |  |
+| `enableAppHealthCheck`   | N        | Enable the app health check on the application | `true`, `false` |
+| `appHealthCheckPath`     | N        | Path to the health check file | `/healthz` |
+| `appHealthProbeInterval` | N        | Interval to probe for the health of the app in seconds
+ |  |
+| `appHealthProbeTimeout`  | N        | Timeout for app health probes in milliseconds |  |
+| `appHealthThreshold`     | N        | Number of consecutive failures for the app to be considered unhealthy |  |
+| `enableApiLogging`       | N        | Enable the logging of all API calls from application to Dapr |  |
+| `env`                    | N        | Map to environment variable; environment variables applied per application will overwrite environment variables shared across applications | `DEBUG`, `DAPR_HOST_ADD` |
+| `appLogDestination`                    | N        | Log destination for outputting app logs; Its value can be file, console or fileAndConsole. Default is fileAndConsole | `file`, `console`, `fileAndConsole` |
+| `daprdLogDestination`                    | N        | Log destination for outputting daprd logs; Its value can be file, console or fileAndConsole. Default is file | `file`, `console`, `fileAndConsole` |
+| `containerImage`| N | URI of the container image to be used when deploying to Kubernetes dev/test environment. | `ghcr.io/dapr/samples/hello-k8s-python:latest`
+| `createService`|  N | Create a Kubernetes service for the application when deploying to dev/test environment. | `true`, `false` |
+
+{{< /table >}}
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
 ## Next steps
 
 Watch [this video for an overview on Multi-App Run](https://youtu.be/s1p9MNl4VGo?t=2456):
```

Comment:
Suggest adding a link to the video for the Kubernetes version also done in the community call.